docs/reference/monitoring/configuring-metricbeat.asciidoc

```diff
@@ -110,6 +115,8 @@ Alternatively, use the
 
 .. Add the `username` and `password` settings to the {es} module configuration
 file. 
+
+.. Add `ssl` related setting. By default ssl is disabled when the ssl section is missing. If Elastic security features are enabled, you must explicitly add {metricbeat-ref}/configuration-ssl.html[ssl section] to allow metricbeat to communicate to the host. 
```

Comment:
```suggestion
.. Add `ssl` settings. By default, TLS/SSL is disabled when these settings are missing. If Elastic {security-features} are enabled, you must explicitly {metricbeat-ref}/configuration-ssl.html[configure SSL] to allow {metricbeat} to communicate to the host. 
```

Reply:
Per @tvernum's comment, I think we should amend this section to say something like:

> If TLS is enabled on the HTTP layer of your Elasticsearch cluster, you must either use `https` as the URL scheme in the `hosts` setting or add the `ssl.enabled: true` setting. Depending on the TLS configuration of your Elasticsearch cluster, you may need to specify {metricbeat-ref}/configuration-ssl.html[additional `ssl.*`] settings.